From 4450fd4a2ad3c5447118d33ee373e555f0e81ab3 Mon Sep 17 00:00:00 2001
From: Laurent Bigonville <bigon@bigon.be>
Date: Thu, 2 Jun 2016 14:41:48 +0200
Subject: [PATCH] Do not start the applet on GNOME and Cinnamon desktops
 (Closes: #40)

{gnome,cinnamon}-settings-daemon are also claiming the D-Bus interfaces
provided by the applet
---
 print-applet.desktop.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/print-applet.desktop.in b/print-applet.desktop.in
index c8fd345..0e38ee1 100644
--- a/print-applet.desktop.in
+++ b/print-applet.desktop.in
@@ -5,6 +5,6 @@ Exec=system-config-printer-applet
 Terminal=false
 Type=Application
 Icon=printer
-NotShowIn=KDE;
+NotShowIn=KDE;GNOME;Cinnamon;
 StartupNotify=false
 X-GNOME-Autostart-Delay=30
